staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -180,6 +188,9 @@ func (s *GenericAPIServer) HealthzChecks() []healthz.HealthzChecker {
 func (s *GenericAPIServer) ListedPaths() []string {
 	return s.listedPathProvider.ListedPaths()
 }
+func (s *GenericAPIServer) OpenAPISpec() *spec.Swagger {
+	return s.OpenAPIService.GetSpec()
```

Comment:
Why do we need this helper? Isn't `s.OpenAPIService.GetSpec` enough? No added value.